### DISCUSSION TITLE
Standard Name Table Release - v87, Nov 2024

### DISCUSSION THREAD

**Discussion by efisher008** [cf-convention/discuss#390] [Announcements relevant to CF]
Dear All,

The standard name table has been updated. The current version is now version 87, dated 12 November 2024. New contributors to the table for this version are Wan-Ling Tseng and Letizia Tedesco. All changes are listed below. Please note that no KWIC index exists yet for this table version, this will be added in due course. At the same time, the Standardized Region List was updated to version 5, dated 12 November 2024.

Best wishes,
Ellie Fisher

**V87 standard name changes:**

**New names**

a. https://github.com/cf-convention/vocabularies/issues/39
`northward_friction_velocity_at_sea_water_surface`
`eastward_friction_velocity_at_sea_water_surface`
Proposed by Aurore Biardeau re. friction velocity in sea water.

b. https://github.com/cf-convention/vocabularies/issues/213
`toa_outgoing_longwave_flux_assuming_clear_sky_and_reference_mole_fraction_of_methane_in_air`
`toa_outgoing_shortwave_flux_assuming_reference_mole_fraction_of_methane_in_air`
`toa_outgoing_shortwave_flux_assuming_clear_sky_and_reference_mole_fraction_of_methane_in_air`
`toa_outgoing_longwave_flux_assuming_reference_mole_fraction_of_methane_in_air`
`reference_mole_fraction_of_methane_in_air`
Proposed by Fiona O'Connor re. methane forcing (for use in CMIP7).

c. https://github.com/cf-convention/vocabularies/issues/214
`surface_downward_mass_flux_of_methane_due_to_soil_biological_consumption`
`tendency_of_atmosphere_mole_concentration_of_ozone_due_to_net_chemical_production`
`atmosphere_absorption_optical_thickness_due_to_nitrate_ambient_aerosol_particles`
`tendency_of_atmosphere_mole_concentration_of_hydroxyl_radical_due_to_chemical_production_from_atomic_singlet_oxygen_and_water_vapor`
`tendency_of_atmosphere_mass_content_of_ammonium_dry_aerosol_particles_due_to_net_chemical_production`
Proposed by Fiona O'Connor re. trace gases and aerosols (for use in AerChemMIP2 and CMIP7).

d. https://github.com/cf-convention/vocabularies/issues/223
`sea_ice_mass_content_of_ice_algae_expressed_as_chlorophyll`
`sea_ice_mole_content_of_ice_algae_expressed_as_carbon`
`gross_primary_productivity_of_biomass_expressed_as_carbon_due_to_ice_algae_in_sea_ice`
`sea_ice_mole_content_of_silicon`
`sea_ice_mole_content_of_nitrate`
Proposed by Letizia Tedesco re. sea ice biogeochemical parameters (for use in CMIP7).

e. https://github.com/cf-convention/vocabularies/issues/224
`tendency_of_mole_concentration_of_particulate_organic_matter_expressed_as_carbon_in_sea_water_due_to_net_primary_production_by_algae`
Proposed by Wan-Ling Tseng re. algal production (for use in CMIP7).

f. https://github.com/cf-convention/vocabularies/issues/229
`volume_fraction_of_pelite_in_sea_floor_sediment`
Proposed by Luke Marsden re. volume fraction of pelite in sea floor sediment.

**P30 Standardized Regions List update to v5, Nov 2024:**

a. https://github.com/cf-convention/vocabularies/issues/64
The following regions were added to the list (proposed by Karl Taylor): 
`northern_hemisphere`
`southern_hemisphere`
If either of these require a description (help text), a separate issue can be opened in this repo for this purpose.

**Comment by japamment** [comment]
Closing this to tidy up previous announcements.
